config/app_local.php.template

```diff
@@ -216,6 +216,8 @@ return [
             'chi' => 'zh-cn',
             'cmn' => 'zh-cn',
             'zh-tw' => ['中文（臺灣正體）', 'zh-hant-tw', 'zh-hk', 'zh-hant-hk'],
+            // Chinese (Wu)
+            'wuu' => [吳語], 'wuu' => 'wu'
```

Comment:
You're missing quotes around `吳語` and you only `'wuu' => ['吳語']`.

The second part (`'wuu' => 'wu'`) should be removed because it would mean you're trying to redirect https://tatoeba.org/wuu to https://tatoeba.org/wu, which is not what we want.

Reply:
Thanks and sorry, Trang. Going to fix it right now,